tests/windows_test.go

```diff
@@ -255,6 +256,55 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 					`DNSDomain[\n\r\t ]+`+expectedSearchDomain+`[\n\r\t ]+`)
 			}, 360)
 		})
+
+		Context("with bridge binding", func() {
+			BeforeEach(func() {
+				By("Starting Windows VirtualMachineInstance with bridge binding")
+				windowsVMI.Spec.Domain.Devices.Interfaces = []v1.Interface{libvmi.InterfaceDeviceWithBridgeBinding(libvmi.DefaultInterfaceName)}
+				windowsVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(windowsVMI)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 420)
+
+				cli = winrnLoginCommand(virtClient, windowsVMI)
+			})
+
+			It("should be recognized by other pods in cluster", func() {
+
+				By("Pinging virt-handler Pod from Windows VMI")
+
+				windowsVMI, err = virtClient.VirtualMachineInstance(windowsVMI.Namespace).Get(windowsVMI.Name, &metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				getVirtHandlerPod := func() (*k8sv1.Pod, error) {
+					winVmiPod := tests.GetRunningPodByVirtualMachineInstance(windowsVMI, windowsVMI.Namespace)
+					nodeName := winVmiPod.Spec.NodeName
+
+					pod, err := kubecli.NewVirtHandlerClient(virtClient).Namespace(flags.KubeVirtInstallNamespace).ForNode(nodeName).Pod()
+					if err != nil {
+						return nil, fmt.Errorf("failed to get virt-handler pod on node %s: %v", nodeName, err)
+					}
+					return pod, nil
+				}
```

Comment:
Looks like we have multiple ways to get the same thing. For example: `getPodByKubeVirtRole`.
But even with this existing method, there is a lot of code duplication as this one is re-defined all over.

Can you please follow up on this and extract it to some reasonable place and update other usages to use it? (in a following PR)